common/ethnicities/wc_ethnicities_gnome.txt

```diff
@@ -0,0 +1,401 @@
+        1 = { name = cheek_height_pos  range = { 0 1 }      }
+		1 = { name = cheek_height_neg  range = { 0 1 }      }
+    }
+	gene_bs_cheek_width = {
+        1 = { name = cheek_width_pos  range = { 0 1 }      }
+    }
+	face_detail_cheek_def = {
+        1 = { name = cheek_def_01  range = { 0 0.4 }      }
+		1 = { name = cheek_def_02  range = { 0 0.4 }      }
+    }
+	face_detail_cheek_fat = {
+        1 = { name = cheek_fat_01_pos  range = { 0 0.5 }      }
+		1 = { name = cheek_fat_02_pos  range = { 0 0.5 }      }
+		1 = { name = cheek_fat_03_pos  range = { 0 0.5 }      }
+		1 = { name = cheek_fat_04_pos  range = { 0 0.5 }      }
+		1 = { name = cheek_fat_01_neg  range = { 0 0.5 }      }
+    }
+	
+	###Chin
+	gene_chin_forward = {
+		1 = { name = chin_forward_neg    range = { 0.4 0.7 }    }
+	}
+	gene_chin_height = {
+        1 = { name = chin_height_pos    range = { 0.4 0.6 }    }
+	}
+	gene_chin_width = {
+        1 = { name = chin_width_pos    range = { 0.35 0.6 }    }
+	}
+	face_detail_chin_def = {
+        1 = {  name = chin_def             range = { 0 1 }     }
+	}
+	face_detail_chin_cleft = {
+        1 = {  name = chin_cleft             range = { 0 0.5 }     }
+	}
+	
+	###Jaw
+	gene_jaw_angle = {
+		1 = { name = jaw_angle_neg    range = { 0.45 0.65 }    }
+	}
+	gene_jaw_forward = {
+		1 = { name = jaw_forward_neg    range = { 0.35 0.70 }    }
+	}
+	gene_jaw_height = {
+		1 = { name = jaw_height_pos    range = { 0.4 0.6 }    }
+	}
+	gene_jaw_width = {
+		1 = { name = jaw_width_pos    range = { 0.6 0.7 }    }
+	}
+	gene_bs_jaw_def = {
+		1 = { name = jaw_def_pos    range = { 0 0.6 }    }
+	}
+	
+	###Mouth
+	gene_mouth_corner_depth = {
+		1 = { name = mouth_corner_depth_neg    range = { 0 0.7 }    }
+	}
+	gene_mouth_corner_height = {
+		1 = { name = mouth_corner_height_pos    range = { 0.45 0.8 }    }
+	}
+	gene_mouth_forward = {
+		1 = { name = mouth_forward_neg    range = { 0.4 0.7 }    }
+	}
+	gene_mouth_height = {
+		1 = { name = mouth_height_pos    range = { 0.6 0.8 }    }
+	}
+	gene_mouth_width = {
+		1 = { name = mouth_width_neg    range = { 0.15 0.75 }    }
+	}
+	gene_mouth_open = {
+		1 = { name = mouth_open_pos    range = { 0 0 }    }
+	}
+	
+	#Philtrum
+	gene_bs_mouth_philtrum_def = {
+		1 = { name = mouth_philtrum_def_pos    range = { 0 1 }    }
+	}
+	gene_bs_mouth_philtrum_shape = {
+		1 = { name = mouth_philtrum_shape_pos    range = { 0 0.5 }    }
+	}
+	gene_bs_mouth_philtrum_width = {
+		1 = { name = mouth_philtrum_width_pos    range = { 0 1 }    }
+		1 = { name = mouth_philtrum_width_neg    range = { 0 1 }    }
+	}
+	
+	#Upper Lip
+	gene_mouth_upper_lip_size = {
+		1 = { name = mouth_upper_lip_size_neg    range = { 0.2 0.6 }    }
+	}
+	gene_bs_mouth_upper_lip_def = {
+		1 = { name = mouth_upper_lip_def_pos    range = { 0 1 }    }
+	}
+	gene_bs_mouth_upper_lip_full = {
+		1 = { name = mouth_upper_lip_full_pos    range = { 0 1 }    }
+		1 = { name = mouth_upper_lip_full_neg    range = { 0 0.65 }  z  }
```

Comment:
```suggestion
		1 = { name = mouth_upper_lip_full_neg    range = { 0 0.65 }    }
```

Reply:
Fixed in a0bfdc5d1cab597195c6649286e610261b73135e!